Tools/Scripts/webkitpy/port/device_port.py

```diff
@@ -69,11 +69,8 @@ def child_processes(self):
         return int(self.get_option('child_processes'))
 
     def driver_name(self):
-        if self.get_option('driver_name'):
-            return self.get_option('driver_name')
-        if self.get_option('webkit_test_runner'):
-            return 'WebKitTestRunnerApp.app'
-        return 'DumpRenderTree.app'
+        parent = super(DevicePort, self).driver_name()
+        return dict(WebKitTestRunner='WebKitTestRunnerApp.app').get(parent, f'{parent}.app')
```

Comment:
This is the same as `f'{parent}.app`, given the only special case (via a dict? why?) does exactly the same thing.